terminate called after throwing an instance of 'std::length_error'
  what():  basic_string::_S_create

This application has requested the Runtime to terminate it in an unusual way.
Please contact the application's support team for more information.
